presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -1288,6 +1271,30 @@ private void testInsertPartitionedBucketedTableWithUnionAll(HiveStorageFormat st
         assertFalse(getQueryRunner().tableExists(getSession(), tableName));
     }
 
+    @Test
+    public void testInsertTwiceToSamePartitionedBucket()
```

Comment:
testInsertIntoPartitionedBucketedTable?

Reply:
there are tests that are testing the single insert to a partitioned bucket table, here the idea is to test the case where the bucket already exists, that's why the name of the test mentions it.